src/runtime/components/ScriptYouTubePlayer.vue

```diff
@@ -120,7 +120,9 @@ const rootAttrs = computed(() => {
   }) as HTMLAttributes
 })
 
+const fallbackPlaceHolder = computed(() => `https://i.ytimg.com/vi/${props.videoId}/hqdefault.jpg`)
 const placeholder = computed(() => `https://i.ytimg.com/vi_webp/${props.videoId}/sddefault.webp`)
+const isFallbackPlaceHolder = ref(false)
 
```

Comment:
```suggestion
type YouTubeThumbnailType =
  // 120x90
  '1' | '2' | '3' | 'default' |
  // 320x180
  'mq1' | 'mq2' | 'mq3' | 'mqdefault' |
  // 480x360
  '0' | 'hq1' | 'hq2' | 'hq3' | 'hqdefault' |
  // 640x480
  'sd1' | 'sd2' | 'sd3' | 'sddefault' |
  // 1280x720
  'hq720' |
  // 1920x1080
  'maxresdefault'

function getYouTubeThumbnail(type: YouTubeThumbnailType, webp = false) {
  return `https://i.ytimg.com/${webp ? 'vi_webp' : 'vi'}/${props.videoId}/${type}.${webp ? 'webp' : 'jpg'}`
}
```

Just draft a function to get YouTube thumbnail here. I think we can use this for `placeholder` or provide for user to use?